pkg/cmd/server/api/types.go

```diff
@@ -491,6 +491,14 @@ type HTPasswdPasswordIdentityProvider struct {
 
 type LDAPPasswordIdentityProvider struct {
 	api.TypeMeta
+	// ClientConfig is the configuration for the LDAP client connection
+	LDAPClientConfig
+	// Attributes maps LDAP attributes to identities
+	Attributes LDAPAttributeMapping
+}
+
+// LDAPClientConfig holds information for connecting to an LDAP server
+type LDAPClientConfig struct {
 	// URL is an RFC 2255 URL which specifies the LDAP search parameters to use. The syntax of the URL is
 	//    ldap://host:port/basedn?attribute?scope?filter
 	URL string
```

Comment:
A search URL doesn't seem very reusable for a client… when you're using this for sync, you'll use several different URL/queries… why does this belong to the client config?